content/pages/faq.md

```diff
@@ -31,8 +31,8 @@ Our women-only events are not publicly advertised and we always specify
 
 ## How can I get involved? ##
 
-Please see the **Volunteer Opportunities** section on our [About
-page]({filename}/pages/about.md)!
```

Comment:
I think we could also quite reasonably keep this text if we want to refer to the about page for a short summary, and the volunteer page for a long-form one.